packages/content/src/graphql/queries/CourseSlug.graphql

```diff
@@ -0,0 +1,9 @@
+query CourseGroupBySlug($Slug: Slug!) {
```

Comment:
@AlexONeillTI Need to fix the `$Slug` here as well. Pretty sure fixing it here and regenerating would've fixed the issue as well. Also, may want to rename the files to something more descriptive like `CourseGroupBySlug.graphql` and `CourseGroupBySlug.generated.tsx`